src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchOptions.cs

```diff
@@ -26,16 +24,30 @@ internal class FindReferencesSearchOptions
         /// </summary>
         public bool AssociatePropertyReferencesWithSpecificAccessor { get; }
 
+        /// <summary>
+        /// Exclude references in unchangeable documents from search results.
+        /// i.e. when a document's corresponding <see cref="IDocumentOperationService.CanApplyChange()"/> returns <c>false</c>.
+        /// </summary>
+        public bool IgnoreUnchangeableDocuments { get; }
```

Comment:
wait.. what?  That seems very strange.  why would FindSymbols ever care about this?  This feels like something a feature+file-changer needs to care about.    Having this baked into this core symbolic service just seems so wacky to me.

Reply:
Oops. The change in find reference is the approach I used in another PR (#38070), which we already abandoned. I probably made a mistake when cherry-picking some test changes from that branch...

I have reverted it from this PR. Thanks!

Reply:
cool thanks :)